Gemfile

```diff
@@ -1,35 +1,24 @@
 source 'https://rubygems.org'
-ruby '2.3.1'
+ruby '2.6.3'
```

Comment:
can we use `~>` ? https://andycroll.com/ruby/use-loose-ruby-versioning-in-your-gemfile/

Reply:
yes